security_privacy_ln.asciidoc

```diff
@@ -0,0 +1,614 @@
+It looks decentralized and non-hierarchical: every node seems to have equal importance.
+Additionally, random graphs have a large diameter.
+In particular, routing in such graphs is challenging as the shortest path between any two nodes is moderately long.
+
+However, in stark contrast, the LN graph is completely different.
+
+=== Lightning graph today
+Lightning is a financial network.
+Thus, the growth and formation of the network are also influenced by economic incentives.
+Whenever a node joins the LN, it may want to maximize its connectivity to other nodes in order to increase its routing efficiency.
+Initially, many Lightning clients were favoring nodes with high degrees in channel establishment.
+As a result, it will be even more likely that newly joining nodes will connect to high-degree nodes.
+This phenomenon is called preferential attachment.
+These economic incentives result in a fundamentally different network than a random graph.
+
+Based on snapshots of publicly announced channels, the degree distribution of the LN follows a power-law function.
+In such a graph, the vast majority of nodes have very few connections to other nodes, while only a handful of nodes have numerous connections.
+At a high-level, this graph topology resembles a star: the network has a well-connected core and a loosely connected periphery.
+Networks with power-law degree distribution are also called scale-free networks.
+This topology is advantageous for routing payments efficiently but prone to certain topology-based attacks.
+
+=== Topology-based attacks
+
+An adversary might want to disrupt the Lightning Network.
+Its goal is to dismantle the whole network into many smaller components, making payment routing practically impossible in the whole network.
+A less ambitious, but still malicious and severe goal might be to only take down certain network nodes.
+Such a disruption might occur on the node-level or on the edge-level.
+
+Let's suppose an adversary can take down any node in the LN.
+For instance, it can DDoS them or make them non-operational by any means.
+It turns out that if the adversary chooses nodes randomly, then scale-free networks like the LN are robust against node-removal attacks.
+This is because a random node lies on the periphery with a small number of connections, therefore playing a negligible role in the network's connectivity.
+However, if the adversary is more prudent, it can target the most well-connected nodes.
+Not surprisingly, the LN and other scale-free networks are _not_ robust against targeted node-removal attacks.
+
+On the other hand, the adversary could be more stealthy.
+Several topology-based attacks target a single node or a single payment channel.
+For example, an adversary might be interested in exhausting a certain payment channel's capacity on purpose.
+More generally, an adversary can deplete all the outgoing capacity of a node to knock it down from the routing market.
```

Comment:
How so? Outgoing capacity is only "shifted around" and never really depleted in a sense. Also users are able to disable certain chanenls to protect the outgoing capacity of those channels.